src/components/StoriesGroup/README.md

```diff
@@ -21,6 +22,12 @@ Component for displaying group of stories. It looks like a carousel in a modal w
 | items          | `[StoriesItem](https://github.com/gravity-ui/uikit/tree/main/src/components/Stories#storiesitem-object)[]`         |          |         | Array of items as in `Stories` component                                                   |
 | thumbnailMedia | `[StoriesItemMedia](https://github.com/gravity-ui/uikit/tree/main/src/components/Stories#storiesitemmedia-object)` |          |         | Media content for preview, otherwise first StoriesItem.media from StoriesItem will be used |
 
+### StoriesGroupIndex object
+
+| Field      | Type     | Required |
+| :--------- | :------- | :------- |
+| groupIndex | `Number` |          |
+
```

Comment:
Missing item index and we might wan't to show link to this type it in PropTypes (main table)

Reply:
[f93638d](https://github.com/gravity-ui/uikit/pull/464/commits/f93638da2ad0fda027017995a259c17203fa46a8)